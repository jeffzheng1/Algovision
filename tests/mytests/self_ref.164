def d = {}
d.d = d
d.d.k = 1
print d.k
print d
