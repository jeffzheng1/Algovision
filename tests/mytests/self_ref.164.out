1
{d: self, k: 1}
